SignalMessaging/utils/ConversationSearcher.swift

```diff
@@ -1,19 +1,77 @@
 //
-//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
 //
 
 import Foundation
 import SignalServiceKit
 
+public class SearchResult {
+    public let thread: ThreadViewModel
+    public let snippet: String?
+
```

Comment:
Should we add an optional "message id"?

Reply:
Or rather an optional TODO to add an optional message id?

Reply:
Sure - if that's what makes sense when adding the Contact results.